core/src/main/java/com/oath/oak/BlocksPool.java

```diff
@@ -15,7 +15,7 @@
  * However it makes creation of the first Oak slower. This initialization is thread safe, thus
  * multiple concurrent Oak creations will result only in the one Pool.
  * */
-class BlocksPool implements BlocksProvider, Closeable {
+final class BlocksPool implements BlocksProvider, Closeable {
```

Comment:
Is this 'final' and the following 'static' fields addition, part of the code styling?

Reply:
Yes. The final is because this class has only private constructors, meaning it already orbit inheritance. The checkstyle plugin enforce the final modifier.
Below, I didn't add the static modifier, I only changed the modifiers order to conform with Java's recommended order.
This is also enforced by the plugin.